programs/kernel-selftests/parse

```diff
@@ -321,33 +321,29 @@ class NetStater < Stater
       # empty @test_subcase when get new @test_case
       @test_subcase = nil if @test_script == 'fib_tests.sh'
     when /^#\s+(PASS|SKIP|FAIL): (.*)/
-      if @test_case
-        # selftests: net: vrf_route_leaking.sh
-        #
-        # ###########################################################################
-        # IPv4 (sym route): VRF ICMP error route lookup traceroute
-        # ###########################################################################
-        #
-        # SKIP: Could not run IPV4 test without traceroute
-        stats.add "#{@test_prefix}.#{@test_case}.#{$2}", $1
-      else
-        # selftests: net: netdevice.sh
-        # SKIP: eth0: interface already up
-        @result = $1
-        @test_case = $2
-
-        if @test_case == 'fdb get tests: iproute2 too old'
-          # below rtnetlink.sh's subtest will repeate twice thus cause duplication
-          # SKIP: fdb get tests: iproute2 too old
-          # and it's passed stat will be
-          # PASS: bridge fdb get
-          stats.add "#{@test_prefix}.bridge_fdb_get", @result unless stats.key? "#{@test_prefix}.bridge_fdb_get"
-        else
-          stats.add "#{@test_prefix}.#{@test_case}", $1
-        end
-        @test_case = nil
-        @result = nil
```

Comment:
thanks, the original code resets @test_case which is missed in the patch. Looks this leads to subtle behavior difference, though it can't be detected by current rspec because 04 and 05 are separately. Let me further think of this to see whether this is a real problem.

Reply:
After further check, the code is ok since it doesn't set @test_case in elsif/else side.